spec/unit/utils/encryptable_transfer_spec.rb

```diff
@@ -56,5 +57,67 @@
       expect(received_file.sender_id).to eq(bob.id)
       expect(received_file.encrypted_transfer_password).to be_nil
     end
+
+    it 'Adds (1) to the encryptable name if this name is already taken in inbox folder' do
+      receiver = bob
+      sender = alice
+
+      encryptable_transfer.transfer(encryptable_credentials1, receiver, sender)
+
+      received_encryptable = bob.inbox_folder.encryptables.first
+
+      expect(received_encryptable.encrypted_transfer_password).to be_present
+
+      encryptable_transfer.transfer(encryptable_credentials1, receiver, sender)
+
+      received_encryptable = bob.inbox_folder.encryptables.last
+
+      expect(received_encryptable.encrypted_transfer_password).to be_present
+      expect(received_encryptable.name).to eq('Personal Mailbox (1)')
+    end
+
+    it 'Adds (2) to the encryptable name if name (1) already is taken in inbox folder' do
+      receiver = bob
+      sender = alice
+
+      encryptable_transfer.transfer(encryptable_credentials1, receiver, sender)
+      expect(receiver.inbox_folder.encryptables.count).to eq 1
+
+      encryptable_credentials1.name += ' (1)'
+
+      encryptable_transfer.transfer(encryptable_credentials1, receiver, sender)
+
+      received_encryptable = bob.inbox_folder.encryptables.first
+
+      expect(received_encryptable.encrypted_transfer_password).to be_present
+      expect(received_encryptable.name).to eq(encryptable_credentials1.name)
+
+      encryptable_transfer.transfer(encryptable_credentials1, receiver, sender)
+
+      received_encryptable = bob.inbox_folder.encryptables.last
+
+      expect(received_encryptable.name).to eq('Personal Mailbox (2)')
+    end
+
+    it 'Does not add (1) when name is not exact' do
```

Comment:
what does it mean ... not exact? don't get it :see_no_evil: 

try to create less technical test cases, more from the user's perspective